### PR TITLE
(feat) Set timeoutExceptionThreshold though XMemcachedClientFactoryBean

### PR DESCRIPTION
**Pull request by matejuh**

### CODE REVIEW

File: src/test/resources/applicationContext.xml [hunk 1]
```diff
@@ -1,49 +1,50 @@
...
-		<property name="transcoder">
-			<bean class="net.rubyeye.xmemcached.transcoders.SerializingTranscoder" />
-		</property>
-		<property name="bufferAllocator">
-			<bean class="net.rubyeye.xmemcached.buffer.SimpleBufferAllocator"></bean>
-		</property>
-	</bean>
-</beans>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (matejuh) [inline src/test/resources/applicationContext.xml]: Looks like the file wasn't actually in UTF-8

**Comment by killme2008** [comment]
:+1: 
Thanks for your work.

**Comment by matejuh** [comment]
Thank you too! Can you please release it? We would like to use it as soon as available.

**Comment by killme2008** [comment]
I will release it ASAP in this weekend, thanks.

**Comment by matejuh** [comment]
May I kindly remind you to release the library?

**Comment by killme2008** [comment]
@matejuh  sorry, i forgot, let me do this right now.

**Comment by killme2008** [comment]
@matejuh  2.4.6 released, it may take some time to be synced into maven central repository.

**Pull request merged**